src/constants.js

```diff
@@ -101,19 +111,14 @@ export const TOKEN_INDEX_MASK = 0b01111111
 export const TOKEN_AUTHORITY_MASK = 0b10000000
 
 /**
- * Mask to check if it's token id creation UTXO (last bit indicates it)
- */
-export const TOKEN_CREATION_MASK = 0b00000001
-
-/**
- * Mask to check if it's mint UTXO (second to last bit indicates it)
+ * Mask to check if it's mint UTXO (last bit indicates it)
```

Comment:
Maybe add a link to the RFC for further information.

Reply:
Should we add the current link? The one still in the MR? Or wait if it's merged?

Reply:
In this case, I think we can merge the RFC first (since it is ready or almost ready to be merged).

Reply:
@obiyankenobi do you think we will merge the RFC soon?

Or should I add a #TODO in the code to add this link in the future?

Reply:
There are just 2 discussions open there: https://gitlab.com/HathorNetwork/rfcs/merge_requests/10

Reply:
I've replied in both discussions in the RFC.

Reply:
Added